### ISSUE TITLE
[Bug]: Lockdown Console Errors

### ISSUE THREAD

**Issue by mmansour** [MetaMask/metamask-extension#20937]
### Describe the bug

Getting these consistently in the developer console on the Brave Browser: Release Notes v1.58.124 (Sep 14, 2023)

### Steps to reproduce

1. Install Metamask (Bug happens when Metamask is closed, locked/logged out. 
2. Open Brave
3. Check Console (Developer Console)

### Error messages or log output

```shell
: 

lockdown-run.js:17 Lockdown failed: TypeError: At intrinsics.Object.groupBy expected boolean not function
  at isAllowedPropertyValue (lockdown-install.js:1:53384)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at isAllowedPropertyValue (lockdown-install.js:1:53041)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at lockdown-install.js:1:55523
  at repairIntrinsics (lockdown-install.js:1:144597)
  at lockdown-install.js:1:145462
  at lockdown-run.js:4:3
(anonymous) @ lockdown-run.js:17
Show 1 more frame
Show less
lockdown-more.js:99 Protecting intrinsics failed: ReferenceError: harden is not defined
  at lockdown-more.js:69:13
  at Set.forEach (<anonymous>)
  at protectIntrinsics (lockdown-more.js:44:22)
  at lockdown-more.js:97:5
(anonymous) @ lockdown-more.js:99
Show 1 more frame
Show less
lockdown-run.js:17 Lockdown failed: TypeError: At intrinsics.Object.groupBy expected boolean not function
  at isAllowedPropertyValue (lockdown-install.js:1:53384)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at isAllowedPropertyValue (lockdown-install.js:1:53041)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at lockdown-install.js:1:55523
  at repairIntrinsics (lockdown-install.js:1:144597)
  at lockdown-install.js:1:145462
  at lockdown-run.js:4:3
(anonymous) @ lockdown-run.js:17
Show 1 more frame
Show less
lockdown-more.js:99 Protecting intrinsics failed: ReferenceError: harden is not defined
  at lockdown-more.js:69:13
  at Set.forEach (<anonymous>)
  at protectIntrinsics (lockdown-more.js:44:22)
  at lockdown-more.js:97:5
(anonymous) @ lockdown-more.js:99
Show 1 more frame
Show less
```
```


### Version

Latest

### Build type

None

### Browser

Brave

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

**Comment by manazpr** [comment]
Same issue I'm getting in chrome.

**Comment by anaamolnar** [comment]
Hello, @mmansour. Thank you for reporting this! The team will take a look at this as soon as they can.

**Comment by Kiki-dingqi** [comment]
looking for the solution

**Comment by sntgchns** [comment]
Facing the same issue on Windows 11, Edge latest version and MetaMask Flask latest

**Comment by gustawdaniel** [comment]
the same described here https://stackoverflow.com/questions/77161596/lockdown-failed-typeerror-at-intrinsics-object-groupby-expected-boolean-not-fu

**Comment by atsoc1993** [comment]
This started happening to me today-- I'm on edge, and not using metamask-- using perawallet on Algorand-- any fix yet?

**Comment by etsraphael** [comment]
Do you know if we have a quick solution? This is causing a loop of hard refresh on my app.

**Comment by dcsan** [comment]
how can you update MM chrome ext? it should update itself i thought but
https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn

updated: September 12, 2023

![image](https://github.com/MetaMask/metamask-extension/assets/514002/07556bed-43cd-4f0a-bbb0-9195f8ddc864)

I love how their address is some apartment in brooklyn lol
> 49 Bogart St APT 22

**Comment by anaamolnar** [comment]
Thanks for reporting everyone! We have a fix for this coming in version 11.1.0. I will close this in favor https://github.com/MetaMask/metamask-extension/issues/20847, as it is a duplicate. Thank you!
